main.py

```diff
@@ -4,28 +4,27 @@
 from common.utils import ynab_api
 from common.utils import config_api
 from enums import AccountTypeToProcessor
-from enums import AccountTypes
-from files_processors.raw_files import RawCSVFile
 
-CONFIG_FILE_PATH = "config.json"
+CONFIG_FILE_PATH = "~/code/ynab-csv-converter/config.json"
 VALID_EXTENSIONS = (".csv", ".xls", ".xlsx")
 
 
-
 def process_files():
-    config_file = open(CONFIG_FILE_PATH)
+    config_file = open(os.path.expanduser(CONFIG_FILE_PATH))
     config_data = json.load(config_file)
     accounts = config_data["accounts"]
     transactions = []
     files_added = []
-    for _root, _dirs, files in os.walk("./"):
+    for _root, _dirs, files in os.walk(os.path.expanduser("~/Downloads")):
```

Comment:
maybe /tmp ?

Reply:
Files from the browser are downloaded to the "Downloads" folder eventually